tokio-io/src/_tokio_codec/framed.rs

```diff
@@ -97,6 +97,24 @@ impl<T, U> Framed<T, U> {
         &mut self.inner.get_mut().get_mut().0
     }
 
+    /// Returns a reference to the underlying codec wrapped by
+    /// `Frame`.
+    ///
+    /// Note that care should be taken to not tamper with the underlying codec
+    /// as it may corrupt the stream of frames otherwise being worked with.
+    pub fn get_codec(&self) -> &U {
```

Comment:
Could this be changed to `codec` and `codec_mut`?

I believe that this is more in line with the [API guidelines](https://rust-lang-nursery.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter)

Cheers